siliconcompiler/tools/openroad/openroad.py

```diff
@@ -912,6 +912,16 @@ def _define_ord_params(chip):
              'list of files to use for specifying global connections',
              field='help')
 
+    _set_parameter(chip, param_key='ord_abstract_lef_bloat_factor',
+                   default_value='10',
```

Comment:
What does this default value translate to?
Is it the same default that openroad had before?

Reply:
it's the same ad the OpenROAD default. It's the number routing pitches the abstract generator will bloat the shapes by to attempt to simplify the layout. Unfortunately that is not always perfect and leaves some complex shapes behind.